src/redux/epics/playground-epic.js

```diff
@@ -12,7 +12,8 @@ const testUpload = (action$, store) => {
   return action$.ofType(playgroundActions.TEST_UPLOAD).mergeMap(action => {
     const file = action.payload;
     return Observable.fromPromise(FileProcessor.initializeUpload(file))
-      .map(({ numberOfChunks, handle, fileName, data }) => {
```

Comment:
@EdmundMai does the playground essentially do the same as the tests I wrote? Do you think we should delete it if so?